code/modules/atmospherics/machinery/air_alarm/_air_alarm.dm

```diff
@@ -0,0 +1,529 @@
+#define AALARM_REPORT_TIMEOUT 100
```

Comment:
is this why it's _air_alarm and not just air_alarm

Reply:
no i just like having the most important file show up at the top.

Reply:
also if this is local undef it at the end

Reply:
ok

Reply:
lmao its actually unused.